src/main/java/org/heigit/ohsome/ohsomeapi/output/dataaggregationresponse/groupbyresponse/GroupByObject.java

```diff
@@ -11,16 +13,10 @@
  * RatioGroupByResult}</li>
  * </ul>
  */
+@Getter
+@AllArgsConstructor
 public abstract class GroupByObject {
 
   @ApiModelProperty(notes = "Object on which the results are grouped on", required = true)
-  protected Object groupByObjectValue;
-
-  public GroupByObject(Object groupByObjectValue) {
-    this.groupByObjectValue = groupByObjectValue;
-  }
-
-  public Object getGroupByObject() {
-    return groupByObjectValue;
-  }
+  protected Object groupByObject;
```

Comment:
Why did you rename this variable?

Reply:
I renamed it because Lombok search for "groubByObjectValue" in the json response, which use although "groupByObject". We get a NullpointerException since "groupByObjectValue" is not present in the json response.
The problem with this change is that the variable name is now the same as the class name.

Reply:
I see.. yeah that's one problem. Another one is that the variable name now does not reflect for what this variable actually stands, as this is it's not an object, but just an identifier, e.g. "Heidelberg". But yeah changing it to something like ```groupByObjectId``` would cause a change in the response as well. I think this would still be an improvement though, so I'd suggest to call it then ```groupByObjectId```.

Reply:
Nice, I will change the variable name.